sse2neon.h

```diff
@@ -2023,6 +2023,23 @@ FORCE_INLINE int _mm_movemask_ps(__m128 a)
 #endif
 }
 
+// Compute the bitwise NOT of a and then AND with a 128-bit vector containing
+// all 1's, and return 1 if the result is zero, otherwise return 0.
+// https://software.intel.com/sites/landingpage/IntrinsicsGuide/#text=_mm_test_all_ones
+FORCE_INLINE int _mm_test_all_ones(__m128i a)
+{
+#define ONES32 0xffffffff
```

Comment:
I am not sure defining a macro here is a good idea or not.

Reply:
I am going to use ` ~(uint64_t) 0` to replace the macro